cif_core.dic

```diff
@@ -24542,6 +24542,179 @@ save_atom_type_scat.dispersion_source
 
 save_
 
+save_atom_type_scat.exponential_polynomial_coefs
+
+    _definition.id                '_atom_type_scat.exponential_polynomial_coefs'
+    _definition.update            2023-04-09
+    _description.text
+;
+    The set of polynomial coefficients for generating X-ray scattering factors:
+    [ a_0, a_1, ... , a_N ].
+
+    f(s; Z) = exp(Sum( a~i~ * s^i^), i=0:N))
+
+    where s = sin(theta)/lambda.
+;
+    _name.category_id             atom_type_scat
+    _name.object_id               exponential_polynomial_coefs
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Matrix
```

Comment:
Maybe `List` would be more suitable here?

This comment applies to other places in this PR that use the `Matrix` types as well.

Reply:
Yes. It does.